[2020-12-03 01:30:30 +0000] [4426] [DEBUG] Current configuration:
  config: None
  bind: ['0.0.0.0:8000']
  backlog: 2048
  workers: 1
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /root/Pizzou2/server
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 0
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: None
  default_proc_name: run:app
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f709902b680>
  on_reload: <function OnReload.on_reload at 0x7f709902b7a0>
  when_ready: <function WhenReady.when_ready at 0x7f709902b8c0>
  pre_fork: <function Prefork.pre_fork at 0x7f709902b9e0>
  post_fork: <function Postfork.post_fork at 0x7f709902bb00>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f709902bc20>
  worker_int: <function WorkerInt.worker_int at 0x7f709902bd40>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f709902be60>
  pre_exec: <function PreExec.pre_exec at 0x7f709902bf80>
  pre_request: <function PreRequest.pre_request at 0x7f70990440e0>
  post_request: <function PostRequest.post_request at 0x7f7099044170>
  child_exit: <function ChildExit.child_exit at 0x7f7099044290>
  worker_exit: <function WorkerExit.worker_exit at 0x7f70990443b0>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f70990444d0>
  on_exit: <function OnExit.on_exit at 0x7f70990445f0>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2020-12-03 01:30:30 +0000] [4426] [INFO] Starting gunicorn 20.0.4
[2020-12-03 01:30:30 +0000] [4426] [DEBUG] Arbiter booted
[2020-12-03 01:30:30 +0000] [4426] [INFO] Listening at: http://0.0.0.0:8000 (4426)
[2020-12-03 01:30:30 +0000] [4426] [INFO] Using worker: sync
[2020-12-03 01:30:30 +0000] [4429] [INFO] Booting worker with pid: 4429
[2020-12-03 01:30:31 +0000] [4426] [DEBUG] 1 workers
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
[2020-12-03 01:30:39 +0000] [4429] [DEBUG] GET /produtos
[2020-12-03 01:31:10 +0000] [4426] [CRITICAL] WORKER TIMEOUT (pid:4429)
[2020-12-03 01:31:10 +0000] [4429] [INFO] Worker exiting (pid: 4429)
[2020-12-03 01:31:10 +0000] [4437] [INFO] Booting worker with pid: 4437
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
[2020-12-03 01:31:10 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:31:10 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:31:35 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:31:40 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:31:45 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:31:56 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:32:00 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:32:04 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:32:08 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:32:21 +0000] [4437] [DEBUG] GET /produtos
[2020-12-03 01:32:29 +0000] [4437] [DEBUG] GET /produtos
[2020-12-03 01:32:39 +0000] [4437] [DEBUG] GET /cardapio
[2020-12-03 01:32:47 +0000] [4437] [DEBUG] OPTIONS /auth
[2020-12-03 01:32:47 +0000] [4437] [DEBUG] OPTIONS /insert/venda
[2020-12-03 01:32:47 +0000] [4437] [DEBUG] POST /insert/venda
[2020-12-03 01:33:00 +0000] [4437] [DEBUG] GET /relatorio/vendas
[2020-12-03 01:33:31 +0000] [4426] [CRITICAL] WORKER TIMEOUT (pid:4437)
[2020-12-03 01:33:31 +0000] [4437] [INFO] Worker exiting (pid: 4437)
{'produtos': [{'nome': 'Fanta-Uva', 'qtd': '2', 'preco': 8, 'subtotal': 14, 'categoria': 'bebida'}]}
[2020-12-03 01:33:31 +0000] [4487] [INFO] Booting worker with pid: 4487
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
[2020-12-03 01:33:33 +0000] [4487] [DEBUG] GET /produtos
[2020-12-03 01:34:10 +0000] [4487] [DEBUG] OPTIONS /auth
[2020-12-03 01:34:10 +0000] [4487] [DEBUG] OPTIONS /insert/produto
[2020-12-03 01:34:11 +0000] [4487] [DEBUG] POST /insert/produto
[2020-12-03 01:34:15 +0000] [4487] [DEBUG] GET /produtos
[2020-12-03 01:34:35 +0000] [4487] [DEBUG] OPTIONS /delete/produto/49
[2020-12-03 01:34:36 +0000] [4487] [DEBUG] DELETE /delete/produto/49
[2020-12-03 01:34:37 +0000] [4487] [DEBUG] GET /produtos
[2020-12-03 01:34:54 +0000] [4487] [DEBUG] GET /cardapio
[2020-12-03 01:35:25 +0000] [4426] [CRITICAL] WORKER TIMEOUT (pid:4487)
[2020-12-03 01:35:25 +0000] [4487] [INFO] Worker exiting (pid: 4487)
[2020-12-03 01:35:25 +0000] [4490] [INFO] Booting worker with pid: 4490
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:813: UserWarning: Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. Defaulting SQLALCHEMY_DATABASE_URI to "sqlite:///:memory:".
  'Neither SQLALCHEMY_DATABASE_URI nor SQLALCHEMY_BINDS is set. '
/root/Pizzou2/server/venv/lib/python3.7/site-packages/flask_sqlalchemy/__init__.py:834: FSADeprecationWarning: SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True or False to suppress this warning.
  'SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and '
[2020-12-03 01:35:53 +0000] [4490] [DEBUG] GET /cardapio
[2020-12-03 01:36:20 +0000] [4490] [DEBUG] GET /produtos
[2020-12-03 01:36:58 +0000] [4490] [DEBUG] GET /produtos
[2020-12-03 01:37:24 +0000] [4490] [DEBUG] OPTIONS /update/produto
[2020-12-03 01:37:25 +0000] [4490] [DEBUG] PUT /update/produto
[2020-12-03 01:37:26 +0000] [4490] [DEBUG] GET /produtos
[2020-12-03 01:37:51 +0000] [4490] [DEBUG] GET /
[2020-12-03 01:38:02 +0000] [4490] [DEBUG] GET /cardapio
[2020-12-03 01:39:12 +0000] [4490] [DEBUG] OPTIONS /auth
[2020-12-03 01:39:12 +0000] [4490] [DEBUG] OPTIONS /insert/venda
[2020-12-03 01:39:13 +0000] [4490] [DEBUG] POST /insert/venda
[2020-12-03 01:39:21 +0000] [4490] [DEBUG] GET /cardapio
[2020-12-03 01:39:24 +0000] [4490] [DEBUG] GET /cardapio
[2020-12-03 01:39:33 +0000] [4490] [DEBUG] OPTIONS /insert/venda
[2020-12-03 01:39:33 +0000] [4490] [DEBUG] OPTIONS /auth
[2020-12-03 01:39:34 +0000] [4490] [DEBUG] POST /insert/venda
